core/src/main/java/tc/oc/pgm/PGMConfig.java

```diff
@@ -164,6 +165,8 @@
     this.verboseStats = parseBoolean(config.getString("ui.verbose-stats", "false"));
     this.griefScore =
         parseInteger(config.getString("gameplay.grief-score", "-10"), Range.atMost(0));
+    this.deathTicks =
+        parseInteger(config.getString("gameplay.death-tick", "15"), Range.atLeast(0));
```

Comment:
`death-ticks`, missing an `s`

Reply:
This must be between 0 and under the minimum respawn time which currently in pgm is 20 ticks. If you set it to a higher number and the map has a low respawn time you'd start seeing some bad behaviour where a player may respawn before their death animation has ended.